wagtailio/taxonomy/tests.py

```diff
@@ -0,0 +1 @@
+# Create your tests here.
```

Comment:
Suggestion: If we're not using them, do we need this files here?

Reply:
Thanks, removed in 70deda8